pyrobosim/pyrobosim/core/world.py

```diff
@@ -78,6 +79,9 @@ def __init__(
         self.object_radius = object_radius
         self.set_inflation_radius(inflation_radius)
 
+        # Occupany grid
```

Comment:
```suggestion
        # Occupancy grid
```
I wonder if this should be an argument in the constructor itself to create the occupancy grid? Not sure.